### DISCUSSION TITLE
Keep or revert https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/523

### DISCUSSION THREAD

**Discussion by akurtakov** [eclipse-mylyn/org.eclipse.mylyn#524] [Polls]
In https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/523 some API that has been made internal via https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues/444 is made public again (a very small portion of the whole change). 
Vote ends on May 17th 2024.

**Comment by ruspl-afed** [comment]
@gnl42 @BeckerFrank @merks - you were the most active project committers during the last months. Please make sure that you have voted here, we need to converge before 2024-06 M3.

**Comment by gnl42** [comment]
I don't know the the pros and cons of internal verses public API, other than that there is no way Mylyn or any connector could be built if all the internal API were not there, so why are they internal?

**Reply by akurtakov** [reply]
Wikitext is actually more used as a standalone library (which happened to be developed as subproject of old Maven TLP) than as a connector to Mylyn. As there is quite good separation of internal vs public packages I fully support not exporting the internal package at all even (instead of x-internal) but  the public ones would better stay such. If one looks into https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues/444 he would notice that only a single package(and it's only few classes) per dialect have been exported. The main wikitext bundle has more exported as this is practically the model to be used when using the library, the implementation details (parsing) has been internal.

**Comment by akurtakov** [comment]
@ruspl-afed It's up to you as a PL to decide what/if anything to do now as the voting period (1 week has passed) and M3 is getting close.

**Reply by ruspl-afed** [reply]
@akurtakov thank you for establishing this poll, it was very informative to see how many people care about Mylyn

As a PL I do respect the voice of community, we will keep your PR.
